api/geyser/src/main/java/org/geysermc/geyser/api/item/custom/NonVanillaCustomItemData.java

```diff
@@ -129,13 +136,6 @@ public interface NonVanillaCustomItemData extends CustomItemData {
      */
     boolean isHat();
 
-    /**
-     * Gets if the item is a tool. This is used to set the render type of the item, if the item is handheld.
-     *
-     * @return if the item is a tool
-     */
-    boolean isTool();
```

Comment:
While I'm not aware of anyone using this, we can't break API like this without proper deprecation.